nms/packages/magmalte/app/views/subscriber/SubscriberUtils.js

```diff
@@ -104,3 +107,84 @@ export type SubscriberInfo = {
   apns: Array<string>,
   policies?: Array<string>,
 };
+
+export function validateSubscriberInfo(
```

Comment:
Need the JSDoc annotations here, and mark the return type with Flow annotations

Reply:
Oh sorry I added this function when I was rebasing but it not used anymore, I added another validate subscriber function